cookies/tests/test_operations.py

```diff
@@ -571,3 +606,68 @@ def tearDown(self):
         Resource.objects.all().delete()
         Collection.objects.all().delete()
         Relation.objects.all().delete()
+
+
+class TestConceptSearch(unittest.TestCase):
+    """
+    Class contains unit test cases for :func:`concept_search` in operations
```

Comment:
Use the full qualified name for the function in the sphinx reference

```
:func:`cookies.operations.concept_search`
```

That way sphinx can find it when we build the API docs.